cmake/StlabUtil.cmake

```diff
@@ -159,7 +154,7 @@ function( stlab_detect_main_executor result_var )
 
   if( task_system STREQUAL "libdispatch" )
     set( result "libdispatch")
-  elseif( task_system STREQUAL "emscripten" )
+  elseif( CMAKE_SYSTEM_NAME STREQUAL "Emscripten" )
```

Comment:
The documentation above needs to reflect this new semantics.

Reply:
Done in #479.